### PR TITLE
Setup Continuous Integration service.

### PR DESCRIPTION
**Pull request by dhruvhacks**
All bugs resolved.

### CODE REVIEW

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
Please add email notification on success as well as on failure.

**Review by dhruvhacks — COMMENTED** [review]
Okay sir,
New commit created.

File: .travis.yml [hunk 1]
```diff
@@ -13,3 +13,8 @@ script:
 
 after_success:
   coveralls
+
+notifications:
+  email:
+    on_success: change
+    on_failure: always
```
Comment (RishabhJain2018) [inline .travis.yml]: Please leave a blank line here.

**Comment by RishabhJain2018** [comment]
@dhruvhacks Always mention the **issue number** in the PR like `#100.` Git then automatically relates the PR with the issue.

File: .travis.yml (hunk 1 above)
Comment (dhruvhacks) [reply .travis.yml]: okay sir,

Comment (RishabhJain2018) [reply .travis.yml]: @dhruvhacks Please leave a line here and then commit again.

Comment (dhruvhacks) [reply .travis.yml]: Sorry sir, Added a new  line at the end of file.

**Comment by RishabhJain2018** [comment]
@dhruvhacks @prakharchoudhary also add a `.coveragerc` and `.coveralls`.yml file.

**Comment by RishabhJain2018** [comment]
The `.coveralls.yml` and `.coveragerc` files will be added in the next issue.

**Pull request merged**